tests/pytest/physics_test.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+import os
+
+import pandas as pd
+
+def test_opt_photons_are_produced_by_nest(file_name_nest):
+    """Check that optical photons come from the NEST process, called 'S1'"""
+
+    particles   = pd.read_hdf(file_name_nest, 'MC/particles')
+    opt_photons = particles[particles.particle_name == 'opticalphoton']
```

Comment:
Could you assert also the case of `thermalelectrons`? It's new  of this PR.

Reply:
I can't with the current actions. I should write a new action only for the test, do you think it is worth it?